src/loaders/ImageLoader.ts

```diff
@@ -8,13 +8,9 @@ import Loader from "./Loader";
 
 export default class ImageLoader extends Loader<HTMLImageElement> {
   public static EVENTS = ["load", "error"];
-  public isPreReady() {
```

Comment:
Has the interface changed?

Reply:
@mixed 

loader is a class that can only be used inside, this method is used internally.